src/conduits/java/com/enderio/conduits/common/blockentity/ConduitBundle.java

```diff
@@ -174,6 +175,10 @@ public NodeIdentifier getNodeFor(IConduitType type) {
         return nodes.get(type);
     }
 
+    public void setNodeFor(IConduitType type, NodeIdentifier node) {
+        nodes.put(type, node);
```

Comment:
push IOState into the node here

Reply:
Maybe fixed in [6126651](https://github.com/SleepyTrousers/EnderIO-Rewrite/pull/85/commits/6126651fb869af118dcededec9a4db5bb92684a3)?